### PR TITLE
feature: arbitration front

### PR DESCRIPTION
**Pull request by samb-local**

### CODE REVIEW

File: app/src/ui/components/tradeDetail/TradeActions.vue [hunk 1]
```diff
@@ -11,6 +12,25 @@ const client = useClientStore()
 const isBuyer = computed(() => props.tradeInfo.trade.buyer === props.walletAddress)
 const isSeller = computed(() => props.tradeInfo.trade.seller === props.walletAddress)
 
+const disputeWinnerMessage = computed(() => {
+  if (props.tradeInfo.trade.state === 'settled_for_taker') {
+    return `Dispute settled for taker: ${formatAddress(getTaker())}`
+  } else if (props.tradeInfo.trade.state === 'settled_for_maker') {
+    return `Dispute settled for maker: ${formatAddress(getMaker())}`
+  } else {
```
Comment (davirds) [inline app/src/ui/components/tradeDetail/TradeActions.vue:20]: Does this code need this `else` returning an empty string?

File: app/src/ui/components/tradeDetail/TradeActions.vue [hunk 2]
```diff
@@ -40,6 +60,13 @@ async function refundEscrow(id: string) {
 async function openDispute(id: string) {
   await client.openDispute(id)
 }
+
+async function settleDispute(winner: string) {
+  console.log(client)
```
Comment (davirds) [inline app/src/ui/components/tradeDetail/TradeActions.vue:65]: Can you remove this `console.log`?

File: app/src/ui/components/tradeDetail/TradeActions.vue [hunk 3]
```diff
@@ -40,6 +60,13 @@ async function refundEscrow(id: string) {
...
+  await client.settleDispute(props.tradeInfo.trade.id, winner)
+  console.log(props.tradeInfo.trade.id, winner)
```
Comment (davirds) [inline app/src/ui/components/tradeDetail/TradeActions.vue:67]: Can you remove this `console.log` too?

File: app/src/ui/pages/TradeDetail.vue [hunk 4]
```diff
@@ -13,26 +14,33 @@ import { usePriceStore } from '~/stores/price'
 import { microDenomToDenom } from '~/utils/denom'
 
 const client = useClientStore()
+const { userWallet } = storeToRefs(client)
 const priceStore = usePriceStore()
 const tradeInfo = ref()
-
+const trade = ref()
```
Comment (davirds) [inline app/src/ui/pages/TradeDetail.vue:20]: Is this variable being used?

**Review by davirds — COMMENTED** [review]
The overall looks great, just a few points to add.

File: app/src/ui/pages/TradeDetail.vue (hunk 4 above)
Comment (samb-local) [reply app/src/ui/pages/TradeDetail.vue:20]: It was being used to trick the compiler (ESLint) into not complaining from the bind on the line 100, however the bind was not necessary and removed together with the variable.

File: app/src/ui/components/tradeDetail/TradeActions.vue [hunk 5]
```diff
@@ -13,13 +13,10 @@ const isBuyer = computed(() => props.tradeInfo.trade.buyer === props.walletAddre
...
-    return `Dispute settled for taker: ${formatAddress(getTaker())}`
-  } else if (props.tradeInfo.trade.state === 'settled_for_maker') {
-    return `Dispute settled for maker: ${formatAddress(getMaker())}`
-  } else {
-    return ''
-  }
+  const taker = `taker: ${formatAddress(getTaker())}`
+  const maker = `maker: ${formatAddress(getMaker())}`
+  const winner = props.tradeInfo.trade.state === 'settled_for_taker' ? taker : maker
+  return `Dispute settled for ${winner}`
```
Comment (davirds) [inline app/src/ui/components/tradeDetail/TradeActions.vue:19]: Very elegant... I loved!

**Review by davirds — APPROVED** [review]

**Pull request merged**